### DISCUSSION TITLE
Colorblind theme feature preview typo - "0.4% of woman"

### DISCUSSION THREAD

**Discussion by alechemy** [community/community#15218] [Other Feature Feedback, Questions, & Ideas]
Should be "women".

<img width="887" alt="Screen Shot 2022-04-19 at 12 09 25 PM" src="https://user-images.githubusercontent.com/22478984/164078218-f74993ca-cdac-400f-a952-ff423ff021d0.png">

**Comment by fredster33** [comment]
Fixed, it seems. #15277 was a duplicate of this one.
